x-pack/plugins/security/public/nav_control/nav_control_component.tsx

```diff
@@ -103,44 +183,8 @@ export class SecurityNavControl extends Component<Props, State> {
         closePopover={this.closeMenu}
         panelPaddingSize="none"
       >
```

Comment:
Adding a `buffer={0}` here will make the popover open below the button, regardless of the popover content. (🎩  hat tip @cchaos )